charts/hail-search/templates/_helpers.tpl

```diff
@@ -0,0 +1,51 @@
+{{/*
```

Comment:
a lot of this looks like its identical to the seqr/templates/_helpers - is there any way we could share these helpers instead of duplicating them?

Reply:
yeah... we can investigate.  The functions are literally identical (not copy-pasted, but the result of running `helm create hail-search`).  

I've honestly found how we use `fullname`/`name`/`Release.Name` to be a little confusing, especially since they all resolve to `seqr` in our own deployment.  Some thought is warranted.

Reply:
helper templates are generally per-chart.

Helm does have the concept of a Library chart, which could come into play here to help reduce duplication of templates and other resources: https://helm.sh/docs/topics/library_charts/